src/reference/modules/reference/pages/variables.adoc

```diff
@@ -27,20 +27,6 @@ Variables are also typed and can be:
 
 More over, all variables in Rudder are overridable at execution time, keep in mind that ordering the definition of your variables is important.
 
-== User defined parameters
```

Comment:
i moved it at the end, and renamed it global parameter to match the name in Rudder UI